src/popup/components/TransactionList.vue

```diff
@@ -238,6 +238,7 @@ export default defineComponent({
     });
 
     onMounted(async () => {
+      root.$store.commit('initTransactions');
```

Comment:
I think that initTransaction is handled on account change - you could work on that or wait for SW-620 to be merged